debug_toolbar/panels/templates/panel.py

```diff
@@ -85,7 +85,8 @@ def _store_template_info(self, sender, **kwargs):
 
         # Skip templates that we are generating through the debug toolbar.
         if (isinstance(template.name, six.string_types) and
-                template.name.startswith('debug_toolbar/')):
```

Comment:
By making this configurable, it becomes possible to unsilence (accidentally by deleting the wrong line in your own `SKIP_TEMPLATE_PREFIXES`, or intentionally) the djdt templates themselves. 
Presumably, it's never been requested to be able to do so (or has been rejected for introducing too much configuration when it wasn't previously potentially necessary) so is it worth keeping the hard-coded extant functionality and making the `SKIP_TEMPLATE_PREFIXES` an *additional* set of skips?